src/main/webapp/app/exam/participate/summary/exercises/programming-exam-summary/programming-exam-summary.component.html

```diff
@@ -3,4 +3,7 @@ <h6>
     <span>
         <a href="{{ this.participation?.repositoryUrl }}" target="_blank" rel="noopener noreferrer">{{ this.participation?.repositoryUrl }}</a>
     </span>
+    <br />
+    <span>{{ 'artemisApp.exam.examSummary.lastCommitHash' | translate }}:</span>
+    <span>{{ this.submission.commitHash }}</span>
```

Comment:
- Most of the time, only the first 7 characters are presented. Should we do this here as well?
- We should add a message on the commit has being loaded / requireing a reload?
```suggestion
    <span [title]="this.submission.commitHash">{{ (submission.commitHash | slice:0:7) || ("some.translate.key" | translate) }}</span>
```